modules/combined/tests/contact_verification/patch_tests/plane_3/plane3_template1.i

```diff
@@ -270,7 +270,7 @@
   solve_type = 'PJFNK'
 
   petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
-  petsc_options_value = 'lu     superlu_dist'
+  petsc_options_value = 'lu            superlu_dist'
```

Comment:
Unfortunately no. You'd have to push to Fande's branch or put a PR into Fande's branch that he can merge in.